examples/cpp/infer_multi/infer_multi.cpp

```diff
@@ -0,0 +1,224 @@
+            << std::endl;
+
+  tf::ClientSession session(root, options);
+
+  std::vector<tf::Tensor> outputs;
+  session.Run({R}, &outputs);
+
+  // Print the output
+  std::cout << "Result: " << outputs[0].matrix<float>() << std::endl;
+}
+
+void PrintVersion() {
+  // nGraph Bridge version info
+  std::cout << "Bridge version: " << tf::ngraph_bridge::ngraph_tf_version()
+            << std::endl;
+  std::cout << "nGraph version: " << tf::ngraph_bridge::ngraph_lib_version()
+            << std::endl;
+  std::cout << "CXX11_ABI Used: "
+            << tf::ngraph_bridge::ngraph_tf_cxx11_abi_flag() << std::endl;
+  std::cout << "Grappler Enabled? "
+            << (tf::ngraph_bridge::ngraph_tf_is_grappler_enabled()
+                    ? std::string("Yes")
+                    : std::string("No"))
+            << std::endl;
+  std::cout << "Variables Enabled? "
+            << (tf::ngraph_bridge::ngraph_tf_are_variables_enabled()
+                    ? std::string("Yes")
+                    : std::string("No"))
+            << std::endl;
+
+  PrintAvailableBackends();
+}
+
+std::unique_ptr<tf::Session> CreateSession(const string& graph_filename) {
+  tf::SessionOptions options;
+  options.config.mutable_graph_options()
+      ->mutable_optimizer_options()
+      ->set_opt_level(tf::OptimizerOptions_Level_L0);
+  options.config.mutable_graph_options()
+      ->mutable_rewrite_options()
+      ->set_constant_folding(tf::RewriterConfig::OFF);
+
+  // The following is related to Grapller - which we are turning off
+  // Until we get a library fully running
+  if (tf::ngraph_bridge::ngraph_tf_is_grappler_enabled()) {
+    options.config.mutable_graph_options()
+        ->mutable_rewrite_options()
+        ->add_custom_optimizers()
+        ->set_name("ngraph-optimizer");
+
+    options.config.mutable_graph_options()
+        ->mutable_rewrite_options()
+        ->set_min_graph_nodes(-1);
+  }
+
+  // Load the network
+  std::unique_ptr<tf::Session> session;
+  tf::Status load_graph_status = LoadGraph(graph_filename, &session, options);
+
+  if (!load_graph_status.ok()) {
+    LOG(ERROR) << load_graph_status;
+    return nullptr;
+  }
+  return std::move(session);
+}
+
+int main(int argc, char** argv) {
+  const char* backend = "CPU";
+
+  if (argc > 1) {
+    backend = argv[1];
+  }
+
+  if (SetNGraphBackend(backend) != tf::Status::OK()) {
+    std::cout << "Error: Cannot set the backend: " << backend << std::endl;
+    return -1;
+  }
+
+  // Run the MatMul example
+  auto session = CreateSession("inception_v3_2016_08_28_frozen.pb");
+
+  // Get the image from disk as a float array of numbers, resized and normalized
+  // to the specifications the main graph expects.
+  std::vector<tf::Tensor> resized_tensors;
+  tf::Status read_tensor_status = ReadTensorFromImageFile(
+      "grace_hopper.jpg", 299 /*input_height*/, 299 /*input_width*/,
+      0.0 /*input_mean*/, 255 /*input_std*/, &resized_tensors);
```

Comment:
Should we park the image in the repo or add instructions how to get the image and frozen graph?

Reply:
Will add instructions.